core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/runtimeModels.scala

```diff
@@ -316,7 +316,7 @@ object RuntimeConfig {
   // Azure machineType maps to `com.azure.resourcemanager.compute.models.VirtualMachineSizeTypes`
   final case class AzureConfig(machineType: MachineTypeName,
                                persistentDiskId: Option[DiskId],
-                               region: com.azure.core.management.Region
+                               region: Option[RegionName]
```

Comment:
oh, I think the original assumption is region comes from user..but I guess it's not true anymore becuz it actually comes from LZ now

cc @cindy-broadinstitute in case this affects ur change

Reply:
Hmm.. does Terra UI actually rely on the runtime region? I see this in my BEE:

<img width="75" alt="image" src="https://github.com/DataBiosphere/leonardo/assets/5368863/d63ef3cc-6656-4ecf-b8c8-c560c7448353">

I'll look at this more, and look into potentially setting the region to the correct value..

Reply:
yea, I think Cindy worked on this recently

Reply:
The terra-ui support for region is in dev, not in production yet.

Reply:
After conversation, I am thinking I will pull the region from WSM from the VM resource, and update the DB in Back Leo. (I haven't pushed that yet.)